src/lib/tools/FileBrowser.svelte

```
@@ -553,33 +551,47 @@
-				if (tree.get_node(parent).original?.folder) {
-					tree.move_node(node, parent, getPositionOfNode(parent, node.text, node.original.folder))
+		jQuery("#" + elemID).on(
+			"move_node.jstree",
+			async (
+				_,
+				{
+					node,
+					parent,
+					old_parent
+				}: {
+					node: any
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)